Resources/Prototypes/Recipes/Reactions/food.yml

```diff
@@ -87,6 +87,19 @@
     - !type:CreateEntityReactionEffect
       entity: FoodCakeBatter
 
+- type: reaction
+  id: CreateTofu
+  impact: Low
+  quantized: true
+  reactants:
+    MilkSoy:
+      amount: 10
+    Enzyme:
+      amount: 5
```

Comment:
Seems to me that from this and the recipes, you intend tofu to just be one item, but on my fork, I was just preparing Tofu as well and intended to make it be made in bigger blocks and sliced for recipes (or to snack on). You might consider doing that, and if you need the sprites for it,

I can arrange PRing it to your branch as I had a community member make some very fine ones for that. I understand it'd mean re-doing a few things, though: Recipe, the item prototype, adding another one, altering the food recipes...

Reply:
![image](https://user-images.githubusercontent.com/117641290/219857643-779780ff-9e4a-4717-970d-8fc889a1d0d7.png)
![image](https://user-images.githubusercontent.com/117641290/219857645-24bc1e6c-e31a-44d2-b7d7-3113797b2118.png)
Preview of the sprites I will use downstream, courtesy of rosysyntax#6514

Reply:
> I can arrange PRing it to your branch

I'd be happy if you do that